distribution/apim-accelerator/bin/merge.sh

```diff
@@ -229,6 +229,15 @@ if [ "${smart_on_fhir_enabled}" == "true" ]; then
       echo -e "\n#[healthcare.identity.claims]\n#patient_id_claim_uri = \"http://wso2.org/claims/patientId\"\n#patient_id_key = \"patientId\"\n#fhirUser_resource_url_context = \"/r4/Patient\"\n#fhirUser_resource_id_claim_uri = \"http://wso2.org/claims/patientId\""  | tee -a "${WSO2_OH_APIM_HOME}"/repository/conf/deployment.toml >/dev/null
   fi
 
+  if grep -Fxq "#[healthcare.backend.auth]" "${WSO2_OH_APIM_HOME}"/repository/conf/deployment.toml || grep -Fxq "[healthcare.backend.auth]" "${WSO2_OH_APIM_HOME}"/repository/conf/deployment.toml
+    then
+        # code if found
+        echo -e "[WARN] healthcare.backend.auth configuration already exist"
+    else
+        # code if not found
+        echo -e "\n[healthcare.backend.auth]\ntoken_endpoint = \"https://localhost:9443/oauth2/token\"\nclient_id = \"client_id\"\nclient_secret = \"client_secret\"\nprivate_key_alias = \"wso2carbon\"\nis_ssl_enabled = true"  | tee -a "${WSO2_OH_APIM_HOME}"/repository/conf/deployment.toml >/dev/null
```

Comment:
cannot use default key alias (wso2carbon) right>?

Reply:
what's the use of is_ssl_enabled?

Reply:
default alias changed in [daa4bf5](https://github.com/wso2/healthcare-accelerator/pull/7/commits/daa4bf5708f8a5dc50a17198b91497494bcd663e).

The external token request can be made with or without `ssl_context`, the config is to toggle that